src/main/java/sirius/biz/storage/util/StorageUtils.java

```diff
@@ -173,6 +176,39 @@ public static String normalizePath(@Nullable String path) {
         return normalizedPath;
     }
 
+    /**
+     * Sanitizes the given path.
+     *
+     * @param path the path to cleanup
+     * @return the sanitized path without backslashes, successive slashes, and without leading and trailing slashes
+     */
+    @Nonnull
+    public String sanitizePath(@Nullable String path) {
+        path = Strings.trim(path);
+
+        if (Strings.isEmpty(path)) {
+            return "";
+        }
+
+        if (path.contains("\\")) {
+            path = path.replace("\\", "/");
```

Comment:
ReplaceAll?

Reply:
replace already replaces each occurrence. The only difference is that replaceAll expects a regex pattern.